scripts/windows/bootstrap.bat

```diff
@@ -55,3 +55,27 @@ rem ==============================
 set "VcPkgDeps=eigen3 suitesparse clapack openblas ceres qt5-base"
 call "%VcPkgDir%\vcpkg.exe" upgrade %VcPkgDeps% --no-dry-run --triplet %VcPkgTriplet%
 call "%VcPkgDir%\vcpkg.exe" install %VcPkgDeps% --triplet %VcPkgTriplet%
+
+
+rem ==============================
+rem Download and build QGLViewer.
+rem ==============================
```

Comment:
```
rem 
rem ... if it does not exist already...?
rem 
set "QGLViewerPath=%~dp0..\..\Thirdparty\libQGLViewer-2.7.1\"
set "QGLVIEWERROOT=%QGLViewerPath%\QGLViewer" 
if not exists "%QGLVIEWERROOT%\QGLViewer2.dll" (
    rem all the statements below...
)
```

Reply:
Maybe even factor out to a separate script e.g. `bootstrap-ogl-viewer.bat` and invoke 
```
call "%~dp0bootstrap-ogl-viewer.bat"
```